Set the default tab globally by using a new DefaultTab ini variable

carlosmauri opened the pull request:
Hi guys,
I've added the ability to set, in a global way, what tab of a content full view should be displayed through a new ini variable located in the ngadminui.ini file.
![image01](https://user-images.githubusercontent.com/25780820/50493322-ba0e3000-09ea-11e9-9e1c-8ec3ec906cb8.jpg)


Right now the default tab we want to display can be set using the DefaultTabs[] array, but we wanted to show the "subitems" tab no matter what's the object class name.

To do so, I've updated the window_controls template to be able to set the $default_tab variable by reading first the new ini file.
`{def $default_tab = ezini( 'ViewSettings', 'DefaultTab', 'ngadminui.ini' )}`

Looking forward to your feedback,
Thank you!

peterkeung commented on bundle/ezpublish_legacy/ngadminui/settings/ngadminui.ini:
```diff
@@ -11,6 +11,12 @@ DefaultTabs[ng_gallery]=subitems
+# The value must be an string.
+# Possible values are: [view|subitems|translations|locations|relations]
+# DefaultTabs[class_identifier] will override this global setting.
+DefaultTab=subitems
```
This should be "view" so we don't break BC.

peterkeung commented on bundle/ezpublish_legacy/ngadminui/design/ngadminui/templates/window_controls.tpl:
```diff
@@ -1,8 +1,12 @@
 {* Window controls *}
+{def $default_tab = ezini( 'ViewSettings', 'DefaultTab', 'ngadminui.ini' )}
+{* If no DefaultTab is defined, display view tab by default *}
```
Do we need this if it's coming from the INI file? Could just look up the INI setting directly in the definitions below.

emodric commented:
Hi @carlosmauri !

Thanks for the contribution :) Wouldn't the default tab still be overriden by class specific configuration? It should be, but is this something you intended to still happen?

What @peterkeung asked is also valid. Is there need to check if the default tab is set in config or not? Can't you just simply load the default value from the INI file?

thiagocamposviana commented:
Also the variable could be defined in the same place as before instead.

carlosmauri commented:
As @peterkeung suggested, I've removed the unnecessary condition since the idea is to have a ini setting in place. I've also moved the variable declaration within the main {def} block. 

Hi @emodric , glad to hear from you. 
The previous logic has not been changed. If a tab value like ```DefaultTabs[folder]``` has been set for a particular content class, it will override the global one.

I've added a comment into the INI file explaining things. Let me know if this is still unclear.
```
# (OPTIONAL) Use this setting to set the default tab of the content full view
# The value must be an string.
# Possible values are: [view|subitems|translations|locations|relations]
# DefaultTabs[class_identifier] will override this global setting.
DefaultTab=view
```

Looking forward to your feedback,
Thanks!

carlosmauri commented:
Sweet! thank you @emodric 🥇
